Services/TMS/Mailing/classes/class.ilTMSMailContextCourseCreation.php

```diff
@@ -0,0 +1,170 @@
+<?php
+declare(strict_types=1);
+
+use \ILIAS\TMS\Mailing;
+use \ILIAS\TMS\CourseCreation\Request;
+
+/* Copyright (c) 2017 Nils Haagen <nils.haagen@concepts-and-training.de> */
```

Comment:
add this to the comment below

Reply:
if it should be the first comment after the open tag, it has to be directly behind the open tag.